data/categories.txt

```diff
@@ -10741,6 +10741,16 @@ sk:Dobíjacia stanica|dobíjanie
 fa:ایستگاه شارژ
 mr:चार्जिंग स्टेशन|प्रभाराक केंद्र
 
+amenity-charging_station|@charging_station
+
+amenity-charging_station-bicycle|@charging_station
+en:4Bicycle Charging
+ru:4Велозарядка
```

Comment:
Translations from string_types.txt should be copied here.

P.S. We should automate it.